site/zenodo_rdm/legacy/deserializers/schemas.py

```diff
@@ -9,10 +9,18 @@
 
 from datetime import date
 
+from flask import current_app
 from invenio_records.dictutils import clear_none
+from marshmallow_utils.schemas import IdentifierSchema
 from nameparser import HumanName
+from werkzeug.local import LocalProxy
 
 from ..vocabularies.licenses import LEGACY_LICENSES, legacy_to_rdm
+from ..vocabularies.utils import _load_json
+
+record_identifiers_schemes = LocalProxy(
```

Comment:
usually this kind of variables go to the config.py, avoiding LocalPorxy here might make it cleaner.

Reply:
in this case `site/zenodo_rdm/config.py`?